### PR TITLE
fix download section

### PR DESCRIPTION
**Pull request by unak**

### CODE REVIEW

**Comment by JuanitoFatas** [comment]
Thanks! This already fixed in #2026 🙇

File: ja/news/_posts/2019-04-01-ruby-2-4-6-released.md [hunk 1]
```diff
@@ -25,21 +25,25 @@ Ruby 2.4.6 がリリースされました。
 ## ダウンロード
 
 * [https://cache.ruby-lang.org/pub/ruby/2.4/ruby-2.4.6.tar.bz2](https://cache.ruby-lang.org/pub/ruby/2.4/ruby-2.4.6.tar.bz2)
```
Comment (JuanitoFatas) [inline ja/news/_posts/2019-04-01-ruby-2-4-6-released.md]: We also use this shorthand syntax for links:

```
<https://cache.ruby-lang.org/pub/ruby/2.4/ruby-2.4.6.tar.bz2>
```

**Comment by unak** [comment]
@JuanitoFatas thank you for fixed English page (and sorry for my late reaction)

File: ja/news/_posts/2019-04-01-ruby-2-4-6-released.md (hunk 1 above)
Comment (unak) [reply ja/news/_posts/2019-04-01-ruby-2-4-6-released.md]: ok

**Comment by JuanitoFatas** [comment]
Thanks 🙇 🙇 🙇 

🌸Happy Friday 🎉

**Pull request merged**